sql/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -89,10 +92,67 @@ public void testTablesAreSwitchedIfLeftIsSmallerThanRight() throws Exception {
 
         operator = JoinPlanBuilder.createNodes(mss, mss.where(), subqueryPlanner, SessionContext.create())
             .build(tableStats, Collections.emptySet());
-        nl = (NestedLoop) operator.build(context, projectionBuilder, -1, 0, null, null, Row.EMPTY, emptyMap());
+        nl = (Join) operator.build(context, projectionBuilder, -1, 0, null, null, Row.EMPTY, emptyMap());
         assertThat(
             ((Collect) nl.left()).collectPhase().distributionInfo().distributionType(),
             is(DistributionType.SAME_NODE)
         );
     }
+
+    @Test
+    public void testHashJoin() throws Exception {
```

Comment:
maybe the method name can be a bit more concrete? what about HashJoin is tested?